### ISSUE TITLE
link between tours

### ISSUE THREAD

**Issue by nvki** [mpetroff/pannellum#844]
Is it possible to set up a hotspot to dynamically switch .json config files in order to link multiple tours together? I have ~100 equipanos for every room (8-12) in every flat (10) of a house and want to make a tour of that.
Having all of the scenes/hotspots defined in a single .json file would be a mess so I'd prefer to create one tour per flat and then link those together. Using URL hotspots works, but without transitions. I've noticed addScene in the api docs but I wanted to check if theres a way to do this oob.

**Comment by mpetroff** [comment]
No, the only way to do this would be to use the `addScene` method.

**Comment by nvki** [comment]
sorry to bother you again - I am failing trying to load scene(s) from a  .json file with the addScene function. Seemingly there is no predefined code that would handle loading an external config file, so should I read it with jQuery?
This is my index.html so far:
`<!DOCTYPE HTML>
<html>
<head>
    <title>tour_test</title>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
    <link type="text/css" rel="Stylesheet" href="../src/pannellum.css"/>
    <link type="text/css" rel="Stylesheet" href="../src/standalone.css"/>
    <script type="text/javascript" src="../src/libpannellum.js"></script>
    <script type="text/javascript" src="../src/pannellum.js"></script>

    <style>
        #panorama {
            height: 90%;
            width: 90%;
        }
        #controls {
            position: absolute;
            bottom: 0;
            z-index: 2;
            text-align: center;
            width: 100%;
            padding-bottom: 3px;
        }
        .ctrl {
            padding: 8px 5px;
            width: 30px;
            text-align: center;
            background: rgba(200, 200, 200, 0.8);
            display: inline-block;
            cursor: pointer;
        }
    </style>

</head>
<body>
    <div id = "panorama">
        <div id = "controls">
            <div class="ctrl" id="overview_btn">&#9650;</div>
        </div>
        <script>
            var viewer = pannellum.viewer("panorama", {
                'scenes': {},
                "author": "Nuki",
                "autoLoad" : false,
                "sceneFadeDuration": 1000,
                "showControls": true,
                "compass": true
            });


            viewer.addScene('map', {
                "type": "equirectangular",
                "panorama": "pano.jpg"
            });

            viewer.loadScene("map");

            document.getElementById('overview_btn').addEventListener('click', function(e) {
                viewer.addScene("test", "tour.json");
                viewer.loadScene("test");
            });
        </script>
    </div>
</body>
</html>`

**Comment by mpetroff** [comment]
The `addScene` function takes an object, not a URL, so if you have a JSON file, you'll need to load it yourself first. If you're already using jQuery, you can load it with that; otherwise, I'd just load it with a standard `XMLHttpRequest`.
